onchain-runtime-wasm/src/primitives.rs

```diff
@@ -141,6 +141,19 @@ pub fn runtime_coin_commitment(coin: JsValue, recipient: JsValue) -> Result<JsVa
     Ok(to_value(&AlignedValue::from(coin.commitment(&recipient)))?)
 }
 
+#[wasm_bindgen(js_name = "runtimeCoinNullifier")]
+pub fn runtime_coin_nullifier(coin: JsValue, sender_evidence: JsValue) -> Result<JsValue, JsError> {
+    let coin: AlignedValue = from_value(coin)?;
+    let sender_evidence: AlignedValue = from_value(sender_evidence)?;
+    let coin = coin_structure::coin::Info::try_from(&**(AsRef::<Value>::as_ref(&coin)))?;
```

Comment:
you can get the same with: `let coin = coin_structure::coin::Info::try_from(&**coin.value.as_ref())?;`
but it's a matter of taste :)